buildtest/cli/__init__.py

```diff
@@ -83,14 +84,14 @@ def positive_number(value):
     return int_val
 
 
-def supported_color(value):
-    """Checks if input is positive number and returns value as an int type.
+def supported_color(input_color):
+    """Checks if input is a supported color and returns value as an Color type.
 
     Args:
-        value (str): Specify an input number
+        value (str): Specify an input color
```

Comment:
please change `value` to `input_color` these args in docstring need to match the method signature.

Reply:
changed